common/dconfig/configupdater.go

```diff
@@ -0,0 +1,28 @@
+package dconfig
+
+import (
+	"github.com/uber/cherami-server/common"
+)
+
+// ConfigUpdater is a daemon that can be used to update the config based on information from some external source
+type (
+	ConfigUpdater interface {
```

Comment:
based on the comment above, this interface / impl might belong to common instead of dconfig.